spec/components/post_destroyer_spec.rb

```diff
@@ -657,7 +657,7 @@
 
     it "should not send the flags_agreed_and_post_deleted message if it was deleted by system" do
       second_post.expects(:update_flagged_posts_count)
-      PostDestroyer.new(Discourse.system_user, second_post).destroy
+      PostDestroyer.new(Discourse.system_user, second_post, { agree_flags: true }).destroy
```

Comment:
If it's the last argument in ruby, you don't need the `{` and `}` - I wonder why rubocop didn't catch this actually.

Reply:
I am experimenting with different code editors for the moment which didn't had rubocop configured. I caught this (on manual rubocop run) after I created this PR.